include/units.h

```diff
@@ -2703,7 +2703,6 @@ namespace units
 
 		template<typename... T>
 		inline constexpr bool has_decibel_scale_v = has_decibel_scale<T...>::value;
-	}

```

Comment:
This `}` should be there to close the namespace. Was there a conflict with v3.x that prevented fast-forwarding? Otherwise, I think rebasing is the better option, as it marks conflicts like these to be solved.